ui-grid/ui-grid.d.ts

```diff
@@ -2087,6 +2087,25 @@ declare module uiGrid {
              * @default true
              */
             enableInfiniteScroll?: boolean;
+            /**
```

Comment:
@joeskeen You are right...I've moved it out into the global grid options. I've committed my changes to my branch but they aren't showing up in the PR yet. I'm new to Github so whats the process now? Should I submit a separate PR?

Thanks for the review and thanks in advance for the help! :)